packages/connect/src/SmoldotProvider/SmoldotProvider.ts

```diff
@@ -101,8 +101,8 @@ export class SmoldotProvider implements ProviderInterface {
   healthPingerInterval = CONNECTION_STATE_PINGER_INTERVAL;
 
    /**
-   * @param {string}   chainSpec  The chainSpec for the WASM client
-   * @param {any}      sm         Optional (only used for testing) defaults to the actual smoldot module
+   * @param string -  chainSpec  The chainSpec for the WASM client
+   * @param any    -  sm         Optional (only used for testing) defaults to the actual smoldot module
```

Comment:
Remove type and "Optional" as these are inferred from the source by tsdoc
```suggestion
   * @param sm - (only used for testing) defaults to the actual smoldot module
```